File: src/UniverseBuilder.Core/Models/Universe.cs
using System;

namespace UniverseBuilder.Core.Models
{
    public class Universe
    {
        public Guid Id { get; set; } = Guid.NewGuid();
        public string Name { get; set; } = string.Empty;
        public string Description { get; set; } = string.Empty;
        public DateTime CreatedDate { get; set; } = DateTime.UtcNow;
        public DateTime ModifiedDate { get; set; } = DateTime.UtcNow;
        public List<EntityMetaData> MetaDataItems { get; set; } = [];
    }
}

File: src/universe-builder-ui/src/types/Universe.ts
export interface Universe {
  id: string;
  name: string;
  description: string;
  createdDate: string;
  modifiedDate: string;
}

export interface CreateUniverseDto {
  name: string;
  description: string;
}

export interface UpdateUniverseDto {
  name: string;
  description: string;
}
